### DISCUSSION TITLE
miscellany/coding/20210604-scope-of-if-statement

### DISCUSSION THREAD

**Discussion by giscus** [kands-code/kands-code.github.io#79] [General]
# miscellany/coding/20210604-scope-of-if-statement

找到属于自己的光

https://blogs.qarks.top/miscellany/coding/20210604-scope-of-if-statement.html

**Comment by kands-code** [comment]
不论什么编程语言，都是写给人看的，所以规范和可读性优先级应该是最高的，除非特别需要性能优化

**Reply by kands-code** [reply]
💫
꒰ঌ(🌸>ワ<)໒꒱：XD
